index.js

```diff
@@ -29,6 +29,9 @@ app.use("/fonts", express.static("./assets/fonts"))
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(sharify)
+// Inject METAPHYSICS_ENDPOINT to sharify data
+// Todo/Enhancement: inject any 'safe ENV vars?
+sharify.data.METAPHYSICS_ENDPOINT = process.env.METAPHYSICS_ENDPOINT
```

Comment:
We can ( to mirror Force's `setup_sharify` middleware), I may have been getting ahead of myself with worrying about mobile or other far-future things.

In terms of standardizing the interface, shall we do that in a subsequent PR? I think this will suffice for now, but I'm happy to give that a shot here.